Boundaries.geojson

```diff
@@ -433,7 +433,7 @@
-  { "type": "Feature", "properties": { "id": "UEEE", "oceanic": "0", "label_lon": "130.0", "label_lat": "66.25", "region": "EMEA", "division": "VATRUS" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 168.0, 72.5 ], [ 168.0, 70.5 ], [ 165.0, 68.666666 ], [ 162.0, 66.55 ], [ 158.0, 66.55 ], [ 158.0, 65.75 ], [ 149.666667, 64.0 ], [ 145.6, 63.833333 ], [ 145.0, 62.0 ], [ 142.0, 62.0 ], [ 140.466666, 61.15 ], [ 138.166667, 59.75 ], [ 135.333333, 59.166666 ], [ 132.0, 58.0 ], [ 131.666667, 57.266667 ], [ 131.4, 56.916666 ], [ 131.0, 56.166666 ], [ 129.366667, 54.85 ], [ 128.166666, 54.666666 ], [ 124.75, 55.166667 ], [ 122.333333, 55.5 ], [ 119.666666, 56.69 ], [ 119.548333, 57.321666 ], [ 119.335, 58.415 ], [ 114.861666, 59.6 ], [ 113.878333, 59.323333 ], [ 108.781666, 60.708333 ], [ 108.033333, 61.266666 ], [ 107.0, 62.0 ], [ 107.0, 65.5 ], [ 106.916666, 67.0 ], [ 106.5, 69.25 ], [ 108.183333, 70.2 ], [ 112.0, 72.0 ], [ 114.733333, 73.833333 ], [ 114.716667, 75.916667 ], [ 114.716667, 76.333333 ], [ 132.0, 74.333333 ], [ 143.583333, 73.2 ], [ 158.0, 72.0 ], [ 168.0, 72.5 ] ] ] ] } },
+  { "type": "Feature", "properties": { "id": "UEEE", "oceanic": "0", "label_lon": "130.0", "label_lat": "66.25", "region": "EMEA", "division": "VATRUS" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [132.00000003,74.33333336], [143.58333336,73.20000003], [158.00000003,72.00000003], [168.00000003,72.50000003], [165.00000003,68.66666694], [162.00000003,66.55000003], [158.00000003,66.55000003], [158.00000003,65.75000003], [149.66666669,64.00000003], [145.50000003,63.56666669], [145.00000003,62.00000003], [142.00000003,62.00000003], [139.95000003,60.85000003], [138.16666669,59.75000003], [135.33333336,59.16666669], [132.00000003,58.00000003], [131.66666669,57.26666669], [131.40000003,56.91666669], [131.00000003,56.16666669], [129.38333336,54.85000003],  [128.16666669,54.66666669],  [122.33333336,55.50000003], [119.66666669,56.68333336], [119.33333336,58.41666669], [114.86666669,59.60000003], [113.88333336,59.31666669], [108.78333336,60.71666669], [108.03333336,61.26666669], [107.00000003,62.00000003], [107.00000003,65.50000003], [106.91666669,67.00000003], [106.50000003,69.25000003], [112.00000003,72.00000003], [114.73333336,73.83333336], [114.71666669,76.33333336] ] ] ] } },
```

Comment:
the rounding errors are still there, check the first tuple for instance: 132.00000003,74.33333336. 
This can be shortened, few more examples in that line.

Reply:
@theShpuntik